CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 * Add a new base cop class `::RuboCop::Cop::RSpec::Base`. The old base class `::RuboCop::Cop::RSpec::Cop` is deprecated, and will be removed in the next major release. ([@bquorning][])
 * Add support for subject detection after includes and example groups in `RSpec/LeadingSubject`. ([@pirj][])
+* Ignore trailing punctuation in context description prefix. ([@elliterate][])
```

Comment:
An easy fix for red CI: add yourself in the end of the file.

Reply:
Ah, thanks! I just looked at recent changes to the file, which must have been from people with previous involvement. I remember wondering about the empty `[]`. Guess I should have pursued that line of thought a bit further.